test/fixtures/instance_simple/outputs.tf

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "instances_self_links" {
+  description = "List of instance self-links"
+  value       = module.instance_simple.instances_self_links
+}
+
+output "project_id" {
+  description = "The GCP project to use for integration tests"
+  value       = var.project_id
+}
+
+output "service_account" {
+  description = "Service account to attach to the instance"
+  value       = var.service_account
+}
+
+
+
```

Comment:
Lets also remove the old fixture

nit: new lines

Reply:
Removed. But what you're pointing to is a different fixture. This one is called `instance_simple`